xchange-stream-binance/src/main/java/info/bitrich/xchangestream/binance/BinanceStreamingExchange.java

```diff
@@ -110,6 +110,12 @@ public Completable connect(ProductSubscription... args) {
     streamingAccountService = new BinanceStreamingAccountService(userDataStreamingService);
     streamingTradeService = new BinanceStreamingTradeService(userDataStreamingService);
 
+    // We disable live subscription at connection time because we don't want to send a subscribe message through the websocket
+    // (not required), and the websocket is limited to 5 incoming messages per second (cannot subscribe to many streams
+    // at the same time via live Subscribing/Unsubscribing feature).
+    // See https://github.com/binance/binance-spot-api-docs/blob/master/web-socket-streams.md#websocket-limits for more details
+    disableLiveSubscription();
```

Comment:
why is this required? do we not default to false?

Reply:
You are right it's totally useless, I removed this block (the **isLiveSubscriptionEnabled** variable is false by default in BinanceStreamingService).

FYI in this commit I also added a validation if streamingService is null in case a user try to enable live subscription before calling connect.